### ISSUE TITLE
FR: Support for Swift Package Manager

### ISSUE THREAD

**Issue by 3ign0n** [SocketMobile/cocoapods-capture#15]
Apple chose [SwiftPM](https://swift.org/package-manager/) as the default package manager instead of CocoaPods.
Could you please add a manifest file, Package.swift to this project?

**Comment by EricGlaenzer** [comment]
Yes we are aware of SwiftPM but last time we check it didn't support packaging a lib file. Did this change?

**Comment by 3ign0n** [comment]
No.
I think that SwiftPM will not support packaging a lib file in the future.
Additionally, Xcode 11.4 or later stop supporting fat framework including fat library *.a any more.
Instead, Apple introduced XCFramework in WWDC19.

To get an overview of XCFramework, [this blog post](https://pspdfkit.com/blog/2020/supporting-xcframeworks/) is good to read.

See the Apple's official documents for details:

What is an XCFramework?
https://help.apple.com/xcode/mac/11.4/#/dev6f6ac218b

Distributing Binary Frameworks as Swift Packages
https://developer.apple.com/documentation/swift_packages/distributing_binary_frameworks_as_swift_packages

Bundling Resources with a Swift Package
https://developer.apple.com/documentation/swift_packages/bundling_resources_with_a_swift_package

**Comment by EricGlaenzer** [comment]
Yes we are working on a new release of the SDK with xcframework

**Comment by FreshopJosiah** [comment]
Is there an ETA on support for xcframework and SPM?

**Comment by EricGlaenzer** [comment]
We just release a new version of our Capture SDK Cocoapods. Next is the SPM release. Thank you for your patience.
This new version of the Capture SDK now support our new S550 product, NFC Contactless Reader / Writer with NFC tag with large payload.

**Comment by chbeer** [comment]
A simple solution for Swift Package Manager would be to provide a pre-built and ZIPed xcframework and a hash for it. That way, SKTCapture could be added as a binaryTarget with a remote URL as described here: https://developer.apple.com/documentation/swift_packages/distributing_binary_frameworks_as_swift_packages

Would that be something you could provide?

**Comment by ian-sayles** [comment]
Have you now released a version that works with SPM?

**Comment by socketPM** [comment]
@chbeer @3ign0n @ian-sayles @FreshopJosiah It's been a while. But we want to give you guys an update: we recently published the SPM version of the SDK. You can check it out here: https://github.com/SocketMobile/swift-package-capturesdk
